busted test fails on Lua5.5

```
Making target test for debian/lua5.5.dh-lua.conf
# tests
Copying src/lxp/lom.lua src/lxp/threat.lua src/lxp/totable.lua in /build/reproducible-path/lua-expat-1.5.2/5.5-expat for test
********************** lua dynamic (5.5) *********
Test: @@LUA@@ /usr/bin/busted spec/
spec/01-lxp_spec.lua:46: attempt to assign to const variable 'k'
[==========] Running tests from scanned files.
[----------] Global test environment setup.
[----------] Running tests from spec/02-threat_spec.lua
[snip]
[----------] 94 tests from spec/02-threat_spec.lua (84.03 ms total)

[----------] Running tests from spec/03-lom_spec.lua
[snip]
[----------] 52 tests from spec/03-lom_spec.lua (30.24 ms total)

[----------] Running tests from spec/04-totable_spec.lua
[snip]
[----------] 40 tests from spec/04-totable_spec.lua (32.70 ms total)

[----------] Global test environment teardown.
[==========] 186 tests from 3 test files ran. (147.33 ms total)
/usr/share/lua/5.5/busted/outputHandlers/gtest.lua:129: attempt to concatenate a nil value

stack traceback:
        /usr/share/lua/5.5/busted/outputHandlers/gtest.lua:129: in upvalue 'getFileLine'
        /usr/share/lua/5.5/busted/outputHandlers/gtest.lua:145: in upvalue 'getTestList'
        /usr/share/lua/5.5/busted/outputHandlers/gtest.lua:173: in upvalue 'getSummaryString'
        /usr/share/lua/5.5/busted/outputHandlers/gtest.lua:219: in field 'fn'
        /usr/share/lua/5.5/mediator.lua:103: in function </usr/share/lua/5.5/mediator.lua:96>
        (...tail calls...)
        /usr/share/lua/5.5/busted/core.lua:221: in function </usr/share/lua/5.5/busted/core.lua:220>

make[2]: *** [/usr/share/dh-lua/make/dh-lua.Makefile.single:312: test-lua-dynamic-real] Error 1
```

The relevant part is:
```
spec/01-lxp_spec.lua:46: attempt to assign to const variable 'k'
```

From https://www.lua.org/manual/5.5/manual.html#8.1
> The control variable in for loops is read only. If you need to change it, declare a local variable with the same name in the loop body.

--- a/spec/01-lxp_spec.lua
+++ b/spec/01-lxp_spec.lua
@@ -41,6 +41,7 @@ describe("lxp:", function()
 		assert(type(cbs) == "table", "expected arg #1 to be a table")
 		local t = {}
 		for k,v in pairs(cbs) do
+			local k, v = k, v -- loop variables are read-only in 5.5+, copy to locals
 			if type(k) == "number" then
 				assert(type(v) == "string", "array entries must have string values")
 				k = v

--- a/spec/01-lxp_spec.lua
+++ b/spec/01-lxp_spec.lua
@@ -41,7 +41,7 @@ describe("lxp:", function()
 		assert(type(cbs) == "table", "expected arg #1 to be a table")
 		local t = {}
 		for k,v in pairs(cbs) do
-			local k, v = k, v -- loop variables are read-only in 5.5+, copy to locals
+			local k = k
 			if type(k) == "number" then
 				assert(type(v) == "string", "array entries must have string values")
 				k = v

